README.md

```diff
@@ -4,7 +4,7 @@
 
 (aka "Algebraic JavaScript Specification")
 
-<img src="logo.png" width="200" height="200" />
+<img src="logo.png" width="200" height="271" />
```

Comment:
Is this change intentional? Perhaps you meant to update the dimensions of __figures/dependencies.png__ instead?